packages/client/package.json

```diff
@@ -52,14 +52,14 @@
     "@types/sdp-transform": "^2.4.6",
     "@types/ua-parser-js": "^0.7.36",
     "@types/ws": "^8.5.4",
-    "@vitest/coverage-c8": "^0.31.0",
+    "@vitest/coverage-v8": "^0.33.0",
     "dotenv": "^16.3.1",
     "prettier": "^2.8.4",
     "rimraf": "^3.0.2",
     "rollup": "^3.25.1",
     "typescript": "^4.9.5",
     "vite": "^4.3.9",
-    "vitest": "^0.30.1",
-    "vitest-mock-extended": "^1.1.3"
+    "vitest": "^0.33.0",
```

Comment:
I see it happening here as well... For some mysterious reason, whenever I attempted to upgrade `vitest`, sample apps couldn't be built anymore as `vite` stopped working 🤯

Reply:
I needed it for `test.extend` but then got around without using it so I think we can revert these changes for now. :D